src/components/players/PlayerPublicProfile.vue

```diff
@@ -14,19 +14,22 @@
 
     <!-- TODO: move description into a sidebar and add deck-listing that only shows this user's decks -->
 
-    <card-codes :content="description" needs-paragraphs></card-codes>
+    <div>{{ description }}</div>
```

Comment:
Description still needs to be output with the card-codes component (otherwise formatting won't come through).

```suggestion
    <card-codes :content="description" needs-paragraphs></card-codes>
```

Reply:
So, I had issues with several players loading with that and couldn't tell what it did that was helpful. I'll add it back with the caveat of "doesn't work on my box"